src/analytics/Properties.tsx

```diff
@@ -1070,6 +1070,10 @@ interface WalletConnectProperties {
   [WalletConnectEvents.wc_request_deny_error]: WalletConnectRequestDenyProperties & {
     error: string
   }
+
+  [WalletConnectEvents.wc_copy_request_payload]: WalletConnectDefaultProperties & {
+    method: string
```

Comment:
Isn't this the same as the `requestMethod` from `WalletConnectDefaultProperties`?

Reply:
oh `requestMethod` exists on `WalletConnectRequestDefaultProperties` but the type is `WalletConnectDefaultProperties` here....i can change it to WalletConnectRequestDefaultProperties but the extra params didn't seem that useful on first glance 😅

Reply:
updated!